# Description: Don't assert that InitializeICU is only called once. It's not harmful
#  and we do this in Oxide if we delete all contexts and then create a new one
# Author: Chris Coulson <chris.coulson@canonical.com>

diff --git a/base/i18n/icu_util.cc b/base/i18n/icu_util.cc
--- a/base/i18n/icu_util.cc
+++ b/base/i18n/icu_util.cc
@@ -49,17 +49,17 @@
 #define ICU_UTIL_DATA_SHARED_MODULE_NAME "icudt.dll"
 #endif
 #endif
 
 namespace base {
 namespace i18n {
 
 bool InitializeICU() {
-#ifndef NDEBUG
+#if 0
   // Assert that we are not called more than once.  Even though calling this
   // function isn't harmful (ICU can handle it), being called twice probably
   // indicates a programming error.
   static bool called_once = false;
   DCHECK(!called_once);
   called_once = true;
 #endif
 
